book/website/_bibliography/references.bib

```diff
@@ -195,13 +195,18 @@ @article{baker2016reproducibility
 }
 
 @article{Nüst2020docker,
-	author  = {Nuest, Daniel and Sochat, Vanessa and Marwick, Ben and Eglen, Stephen and Head, Tim and Hirst, Tony and Evans, Benjamin},
-	title   = {Ten Simple Rules for Writing Dockerfiles for Reproducible Data Science},
-	journal = {OSF Preprints},
-	doi = {10.31219/osf.io/fsd7t},
+	doi = {10.1371/journal.pcbi.1008316},
+	url = {https://doi.org/10.1371/journal.pcbi.1008316},
 	year = {2020},
-	month = {apr},
-	URL = {osf.io/fsd7t}
+	month = nov,
+	publisher = {Public Library of Science ({PLoS})},
+	volume = {16},
+	number = {11},
+	pages = {e1008316},
+	author = {Daniel N\"{u}st and Vanessa Sochat and Ben Marwick and Stephen J. Eglen and Tim Head and Tony Hirst and Benjamin D. Evans},
```

Comment:
Can we keep the author listing consistent with other entries?
"author  = {N\"{u}st, Daniel and Sochat, Vanessa and Marwick, Ben and Eglen, Stephen J. and Head, Tim and Hirst, Tony and Evans, Benjamin},"

Reply:
Sure!